setup.py

```diff
@@ -164,8 +164,8 @@ def run(self):
   libs.extend(['libonnxruntime_providers_tensorrt.so'])
   # OpenVINO Libs
   if package_name == 'onnxruntime-openvino':
-    if platform.system() == 'Linux':
-      libs.extend(['libovep_ngraph.so'])
+    libs.extend(['libonnxruntime_providers_openvino.so'])
```

Comment:
should we just follow the pattern above with DNNL and TensorRT EP instead? (and add comment)